etcd/etcd.go

```diff
@@ -246,3 +236,29 @@ func (c *Client) Delete(id string) error {
 
 	return err
 }
+
+func (c *Client) UpsertAll(routes eskip.RouteList) error {
```

Comment:
I deal with this method as soon as I start dealing with the delete for innkeeper #81